services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -151,7 +151,7 @@ def create_app():
 
     # also sets up mounted_volumes
     setup_mounted_fs(app)
-    setup_directory_watcher(app)
+    setup_outputs(app)
```

Comment:
minor: to be honnest I found the old name more informative.

Reply:
What previously was called directory watcher is now part of the `outputs` module. I think this is ok.